ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.17.yaml

```diff
@@ -0,0 +1,893 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
I wonder if we need to update the tag here

Reply:
I don't think it would hurt to update it. Before we do that we might also want to update the go version in ocp4e2e, too.

https://github.com/ComplianceAsCode/ocp4e2e/blob/main/go.mod#L3

Reply:
Kicking the tires on that in https://github.com/ComplianceAsCode/ocp4e2e/pull/47